app/controllers/applicants_controller.rb

```diff
@@ -146,35 +146,26 @@ def unassigned
     @applicant_presenter = ApplicantPresenter.new
     applicants = repo.applicant.get_unassigned_unarchived_applicants
     @applicants = @applicant_presenter.sort_by_date(applicants)
-    @craftsmen = Footprints::Repository.craftsman.all
+    @london_crafters = Footprints::Repository.craftsman.where("location = 'London' and seeking = 't'", 0)
```

Comment:
There's probably a better way to access a list for each location without manually defining one for each location. What if another location was added?